File: app/src/main/java/com/formichelli/dfnightselfies/WindowUtil.java
package com.formichelli.dfnightselfies;

import android.app.Activity;
import android.content.Context;
import android.view.Window;
import android.view.WindowManager;

public class WindowUtil {
    public static void setupWindow(Activity activity)
    {
        Window w = activity.getWindow();

        // hide title
        activity.requestWindowFeature(Window.FEATURE_NO_TITLE);

        // hide statusbar if not on lollipop
        if (android.os.Build.VERSION.SDK_INT < 21)
            w.setFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN, WindowManager.LayoutParams.FLAG_FULLSCREEN);

        // keep screen on
        w.addFlags(android.view.WindowManager.LayoutParams.FLAG_KEEP_SCREEN_ON);

        // set brightness to maximum
        WindowManager.LayoutParams windowAttributes = w.getAttributes();
        windowAttributes.screenBrightness = 1;
        w.setAttributes(windowAttributes);
    }
}

File: app/src/main/java/com/formichelli/dfnightselfies/WindowUtil.kt
package com.formichelli.dfnightselfies

import android.app.Activity
import android.os.Build
import android.view.View
import android.view.Window
import android.view.WindowManager
import kotlinx.android.synthetic.main.fragment_dfnightselfies_main.*

object WindowUtil {
    fun setupWindow(activity: Activity) {
        // hide title
        activity.requestWindowFeature(Window.FEATURE_NO_TITLE)

        // hide status bar if not on lollipop
        if (android.os.Build.VERSION.SDK_INT < 21)
            activity.window.setFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN, WindowManager.LayoutParams.FLAG_FULLSCREEN)

        // keep screen on
        activity.window.addFlags(android.view.WindowManager.LayoutParams.FLAG_KEEP_SCREEN_ON)

        // set brightness to maximum
        val windowAttributes = activity.window.attributes
        windowAttributes.screenBrightness = 1f
        activity.window.attributes = windowAttributes
    }

    fun setBackgroundColor(activity: Activity, views: List<View> , color: Int) {
        views.forEach {
            it.setBackgroundColor(color)
        }

        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
            activity.window.statusBarColor = color
            activity.window.navigationBarColor = color
        }
    }

}